app/backend/controllers/login-signup.js

```diff
@@ -59,3 +69,30 @@ module.exports.login = async (request, response) => {
     }
   }
 }
+
+/*
+  Get endpoint for verify email.
+*/
+module.exports.verify = async (request, response) => {
+  const token = request.query.token
+  try {
```

Comment:
`try/catch` structure doesn't fit here. We can handle promise errors manually, instead of relying on a catch-all mechanism.